code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -41,7 +41,7 @@
 	primitive_form = SPECIES_MONKEY_UNATHI
 	darksight = 3
 	ambiguous_genders = TRUE
-	//gluttonous = 1 //VOREStation Edit - Redundant
+	gluttonous = 1
```

Comment:
This is still redundant and should not have just been changed to a 0 in the alt files.

Reply:
That is how Aronai wanted me to do it.

Reply:
Same effect, and that var in particular has caused a merge conflict every time Polaris has changed these files. So now that won't happen anymo'.